examples/ai-core/src/generate-text/google-image-tool-results.ts

```diff
@@ -0,0 +1,85 @@
+import { google } from '@ai-sdk/google';
+import { generateText, stepCountIs, tool } from 'ai';
+import { z } from 'zod/v4';
+import 'dotenv/config';
+import { anthropic } from '@ai-sdk/anthropic';
+
+// Helper function to convert URL to base64
+async function urlToBase64(url: string): Promise<string> {
+  const response = await fetch(url);
+  const arrayBuffer = await response.arrayBuffer();
+  const bytes = new Uint8Array(arrayBuffer);
+  return Buffer.from(bytes).toString('base64');
+}
+
+const imageAnalysisTool = tool({
+  description: 'Give the image ',
+  inputSchema: z.object({}),
+  execute: async ({}) => {
+    try {
+      const base64Image = await urlToBase64(
+        'https://images.unsplash.com/photo-1751225750479-43ad27b94fa0?w=900&auto=format&fit=crop&q=60&ixlib=rb-4.1.0&ixid=M3wxMjA3fDB8MHxmZWF0dXJlZC1waG90b3MtZmVlZHwyfHx8ZW58MHx8fHx8fHx8',
+      );
```

Comment:
could you use a local test file instead? We have https://github.com/vercel/ai/blob/e83bfe37fbed408c7af127d5a256f82ea8e36fe9/examples/ai-core/data/comic-cat.png